roles/klaytn_grafana/tasks/main.yml

```diff
@@ -1,19 +1,67 @@
 ---
-# tasks file for klaytn_bridge
+# tasks file for klaytn_grafana
+- name: "kcnd - define Klaytn service"
+  set_fact:
+    klaytn_service_type: kcnd
+  when: inventory_hostname.find('CN') != -1
+
+- name: "kpnd - define Klaytn service"
+  set_fact:
+    klaytn_service_type : kpnd
+  when: inventory_hostname.find('PN') != -1
+
+- name: "kend - define Klaytn service"
+  set_fact:
+    klaytn_service_type : kend
+  when: inventory_hostname.find('EN') != -1
+
+- name: "kscnd - define Klaytn service"
+  set_fact:
+    klaytn_service_type: kscnd
+  when: inventory_hostname.find('SCN') != -1
+
+- name: "kspnd - define Klaytn service"
+  set_fact:
+    klaytn_service_type : kspnd
+  when: inventory_hostname.find('SPN') != -1
+
+- name: "ksend - define Klaytn service"
+  set_fact:
+    klaytn_service_type : ksend
+  when: inventory_hostname.find('SEN') != -1
+
 - name: "grafana"
   set_fact:
     isGrafana: "yes"
     prometheus_port: "{{ prometheus_port }}"
     grafana_port: "{{ grafana_port }}"
+    influxdb_port: "{{ influxdb_port }}"
   when: inventory_hostname.find('GRAFANA') != -1
 
+- name: "Klaytn nodes"
+  set_fact:
+    isGrafana: "no"
+  when: inventory_hostname.find('GRAFANA') == -1
+
 - name: "Set Ansible values"
   set_fact:
     grafana_conf_dir: "/etc/grafana/conf"
   when:
     - isGrafana|bool
 
+- name: "Set Ansible values"
```

Comment:
The name of the rule seems not correct.

Reply:
Well, the task does set some values (e.g., `/etc/telegraf`) for some keys (e.g., `telegraf_conf_dir`)...  any suggestions?

Reply:
The two `Set Ansible values` tasks set variables for grafana and telegraf file locations, but the name seems ambiguous. How about "Set grafana conf dir" and "Set telegraf conf and klaytn log dirs"?